core/src/main/java/org/mskcc/cbio/portal/scripts/ParquetWriter.java

```diff
@@ -0,0 +1,126 @@
+/*
```

Comment:
As previously noted,  I would move this to a util package in the persistence-spark